examples/federation/base-app/src/main/kotlin/com/expediagroup/graphql/sample/RoutesConfiguration.kt

```diff
@@ -16,69 +16,23 @@
 
 package com.expediagroup.graphql.sample
 
-import com.expediagroup.graphql.extensions.print
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.type.MapType
-import com.fasterxml.jackson.databind.type.TypeFactory
-import graphql.schema.GraphQLSchema
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.core.io.Resource
-import org.springframework.http.HttpMethod
-import org.springframework.http.MediaType
-import org.springframework.web.reactive.function.server.ServerRequest
-import org.springframework.web.reactive.function.server.bodyToMono
+import org.springframework.web.reactive.function.server.bodyAndAwait
+import org.springframework.web.reactive.function.server.coRouter
 import org.springframework.web.reactive.function.server.html
-import org.springframework.web.reactive.function.server.json
-import org.springframework.web.reactive.function.server.router
-import reactor.core.publisher.Mono
 
 @Configuration
 class RoutesConfiguration(
-    val schema: GraphQLSchema,
-    private val queryHandler: QueryHandler,
-    private val objectMapper: ObjectMapper,
     @Value("classpath:/graphql-playground.html") private val playgroundHtml: Resource
 ) {
 
-    private val mapTypeReference: MapType = TypeFactory.defaultInstance().constructMapType(HashMap::class.java, String::class.java, Any::class.java)
-
-    @Bean
-    fun graphQLRoutes() = router {
-        (POST("/graphql") or GET("/graphql")).invoke { serverRequest ->
-            createGraphQLRequest(serverRequest)
-                .flatMap { graphQLRequest -> queryHandler.executeQuery(graphQLRequest) }
-                .flatMap { result -> ok().json().syncBody(result) }
-                .switchIfEmpty(badRequest().build())
-        }
-    }
-
-    @Bean
-    fun sdlRoute() = router {
-        GET("/sdl") {
-            ok().contentType(MediaType.TEXT_PLAIN).syncBody(schema.print())
-        }
-    }
-
     @Bean
-    fun graphQLToolRoutes() = router {
+    fun playgroundRoute() = coRouter {
```

Comment:
Do we want to include the playground in the library? Everyone is going to include graphiql or the playground if we don't so we could just add this in.

Reply:
while certainly useful, `playground` is optional and you don't need it to run a GraphQL server. I removed it from `spring-server` due to the above and also to simplify our logic in how to support custom `/graphql` (and `subscriptions` endpoint) as this value is hard coded in the HTML.